internal/store/driver.go

```diff
@@ -642,8 +844,424 @@ func (d *Driver) calculateHTTPSEdges() map[string]ingressv1alpha1.HTTPSEdge {
 			edgeMap[rule.Host] = edge
 		}
 	}
+}
 
-	return edgeMap
+func (d *Driver) calculateHTTPSEdgesFromGateway(edgeMap map[string]ingressv1alpha1.HTTPSEdge) {
+	gateways := d.store.ListGateways()
+
+	for _, gtw := range gateways {
+		for _, listener := range gtw.Spec.Listeners {
+			if listener.Hostname == nil {
```

Comment:
shouldn't this match the logic in https://github.com/ngrok/kubernetes-ingress-controller/pull/343/files#diff-9ece2bb7eeaf621263ba5abe1c62e28734da229e31faf7348e3aa4b260fc73aaR704 ?

Reply:
good catch, i'll add all of these to a follow up ticket